x-pack/plugins/security_solution/public/resolver/store/mocks/related_event.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import uuid from 'uuid';
+import { EndpointEvent } from '../../../../common/endpoint/types';
+
+/**
+ * Simple mock related event.
+ */
+export function mockRelatedEvent({
+  entityID,
+  timestamp,
+  category,
+  type,
+}: {
+  entityID: string;
+  timestamp: number;
+  category: string;
+  type: string;
+}): EndpointEvent {
+  return {
+    '@timestamp': timestamp,
+    event: {
+      kind: 'event',
+      type,
+      category,
+      id: uuid.v4(),

```

Comment:
can you make this non-random? maybe pass it in or just make it `id: '1'` or something

Reply:
https://github.com/elastic/kibana/pull/74411/commits/a48a10a6244e3ea35eda3de095fddb427e645c91#diff-d991ae971eb0dc808e612349a2c70b9aR30